src/Components/Publishing/Sections/Text.tsx

```diff
@@ -33,3 +43,22 @@ export const Text: React.SFC<TextProps> = props => {
 Text.defaultProps = {
   color: "black",
 }
+
+const htmlMaybeWithContentEnd = (html, isContentEnd) => {
+  const doc = document.createElement("div")
+  doc.innerHTML = html
+  // remove existing content-end spans, remove after backfill
```

Comment:
Ah we should address this backfill. Want to put some time in the calendar so I can walk you through it?

Reply:
Yes sounds great, would be glad for a walkthrough of backfilling sections!